h2o-automl/src/main/java/ai/h2o/automl/preprocessing/TargetEncoding.java

```diff
@@ -48,6 +50,9 @@ public void prepare() {
         Frame amlTrain = _aml.getTrainingFrame();
         Set<String> teColumns = selectColumnsToEncode(amlTrain);
         if (teColumns.isEmpty()) return;
+        
+        _aml.eventLog().warn(Stage.FeatureCreation,
+                "Target Encoding integration in AutoML is in experimental stage, the models obtained with this feature can not be downloaded as MOJO for production yet.");
```

Comment:
When running this from the client (this is from R), the print-out of this warning goes very wide and the AutoML status bar gets printed over the top of it (it seems)... can we have multi-line warning messages and/or force a newline before the AutoML status bar gets printed again?  Or is this caused by the fact that AutoML status bar starts, then this TE warning prints, and then it goes back to status bar (and thus we can't fix it...)?

It looks like this:
```
18:59:03.228: Target Encoding integration in AutoML is in an experimental stage, the models obtained with this feature can not yet be downloa  |======================================================================| 100%
Total AutoML runtime (secs): 3341
```

Reply:
it's a display issue with all event logs when the progress bar is on.
I can't do anything immediately, I need to reconsider this globally, for example:
1.  save the progress bar state
2. display event logs
3. restore the progress bar
Would be a fun ticket when I'm bored.

Reply:
Thanks @sebhrusen I created a JIRA here: https://0xdata.atlassian.net/browse/PUBDEV-7802